storage/postgrest/entrypoint.sh

```diff
@@ -1,18 +1,23 @@
 #!/bin/bash
 
 export PGRST_DB_URI="postgres://${DATABASE_USER}:${DATABASE_PW}@${DATABASE_HOST}:${DATABASE_PORT}/${DATABASE_NAME}"
-export PGRST_DB_SCHEMA=storage
 #In production this role should not be the same as the one used for the connection
+export PGRST_DB_SCHEMA=$DATABASE_SCHEMA
 export PGRST_DB_ANON_ROLE="${DATABASE_ANON}"
 
 psql --version
-printf 'Waiting for %s to be ready\n' "${DATABASE_URL}"
+printf 'Waiting for %s to be ready\n' "${DATABASE_HOST}:${DATABASE_PORT}/${DATABASE_NAME}"
 
 (( max_retry=30 ))
 (( current_retry=1 ))
 
+READ_ONLY_COMMAND="GRANT CONNECT ON DATABASE ${DATABASE_NAME} TO ${PGRST_DB_ANON_ROLE};
+GRANT USAGE ON SCHEMA ${PGRST_DB_SCHEMA} TO ${PGRST_DB_ANON_ROLE};
+GRANT SELECT ON ALL TABLES IN SCHEMA ${PGRST_DB_SCHEMA} TO ${PGRST_DB_ANON_ROLE};
+GRANT SELECT ON ALL SEQUENCES IN SCHEMA ${PGRST_DB_SCHEMA} TO ${PGRST_DB_ANON_ROLE};"
```

Comment:
I think this should be done in the liquibase script where the anon user is created. Or is there a specific reason to do it here?

Reply:
It somehow seems not not work with the liquibase script, maybe another postgresScheme error?

Reply:
Ah damn, it seems like, I have to change the storage-mq scheme via a env flag, not this one.